spec/player_spec.rb

```diff
@@ -0,0 +1,29 @@
+require './lib/player'
+
+RSpec.describe Player do
+  context 'Create player instances' do
+    it 'raise an ArgumentError if an integer was given' do
+      expect { Player.new('23', 'X') }.to raise_error(ArgumentError, 'you can\'t have an Integer as name')
+    end
+
+    it 'raise an ArgumentError if a short name was given' do
+      expect { Player.new('zac', 'O') }.to raise_error(ArgumentError, 'A name should at least contain 4 letters')
+    end
+
+    it 'make sure that an object is create with no errors' do
+      expect(Player.new('Javier', 'X')).to be
+    end
+  end
+
+  context '#inc_score' do
```

Comment:
* [ ] Specs for each method should be wrapped in one `describe` block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples . ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/describe.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))

- change the context on line #18 to be a  `describe` block.